src/test/unit/controls/Timeline/Timeline-spec.js

```diff
@@ -235,14 +235,18 @@ describe("Timeline", () => {
             input.content = null;
             data.tasks = [];
             const cfg = timeline.config;
+            const ticks = document.getElementsByClassName("tick");
+            const leftValue = ticks[0].getBoundingClientRect().width / 2;
+            const rightValue =
+                ticks[ticks.length - 1].getBoundingClientRect().width / 2;
```

Comment:
Why are the expected values calculated here? You are using the same formula as used in the source code that is going to work. This is not how you should be testing.

Reply:
Fixed 675cd0b

Reply:
Fixed here e607e6e